perma_web/perma/celery_tasks.py

```diff
@@ -23,6 +23,7 @@
 from django.conf import settings
 from django.utils import timezone
 from django.template.defaultfilters import pluralize
+from django.template.defaultfilters import filesizeformat
```

Comment:
The tiniest of tweaks: since the line above it also importing from `django.template.defaultfilters`, can you combine these two lines?